docs/site/Using-MessagePack-with-LoopBack.md

```diff
@@ -0,0 +1,10 @@
+---
+lang: en
+title: 'How to use MessagePack with LoopBack'
+keywords: LoopBack 4.0, LoopBack 4, MessagePack
+sidebar: lb4_sidebar
+layout: readme
+source: loopback-next
+file: bodyparsers/msgpack/README.md
+permalink: /doc/en/lb4/Using-messagepack-with-loopback.html
```

Comment:
AFAICT, this page is not referenced from the sidebar, can we remove it?

As of https://github.com/strongloop/loopback.io/pull/996, we allow sidebar entries (group headings) with no linked page.

Reply:
It's being referenced here: https://github.com/strongloop/loopback-next/pull/6155/files#diff-967acfe8350a9043a788970b5cf91223R239